dnf/cli/format.py

```diff
@@ -15,8 +15,23 @@
 # Red Hat, Inc.
 
 from __future__ import unicode_literals
+from datetime import datetime
```

Comment:
Do we change laws if there are people that break them?
Anyway, the correct workflow is:
1. find a good reason why current guidelines does not work
2. suggest the change
3. code

not:
1. break it
2. question it